### PR TITLE
Reformat beforeyoufinish_page, replace Android App Icon and update Android app name , fix clientValidateAccessCode code

### PR DESCRIPTION
**Pull request by ianradnor088**
Added Simply white earth styling in some pages, also working on adding Css in global.css file, this things are in process now.

The client asked for changes to the page.  I made them in input.json, but because it used to have

text, then

“text in quotes” then

text

It will not look correct on the screen.  I have attached what is used to look like.  The client has ask for these text changes.

Your ballot is being digitally sealed. Before you submit your ballot, you can perform a check to make sure it is recorded and sealed correctly. How? It's easy. 1. Use a laptop or other device if you have one. If you don't have a second device, open the browser app on this device. 2. Go to VerifyMyBallot.gov. 3. Follow the instructions shown. Want to learn more about ballot checking? Visit electionwebsite.gov. --done

### CODE REVIEW

**Comment by awoflaherty** [comment]
Rename the PR request.  "Done THAJ - 108, 109, 111 and added some files #27"

**Review by awoflaherty — COMMENTED** [review]
@ianradnor088 I don't see any changes to beforeyoufinish_page.  Is this in this PR?

File: scripts/resources.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
+const fs = require('fs');
```
Comment (mattbeattie) [inline scripts/resources.js:1]: What does this script do? When would I run it? Can you add some documentation?

**Comment by awoflaherty** [comment]
Closed because this PR did not comply and was not corrected.

File: src/app/pages/access-code/access-code.page.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable eqeqeq */
```
[the remark this answers is not in the document]
Comment (mattbeattie) [reply src/app/pages/access-code/access-code.page.ts:1]: Update: this is fine for now, we'll investigate in a separate PR.

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "MVP-Test-Harness",
+  "name": "Rosie Prototype",
```
[the remark this answers is not in the document]
Comment (ianradnor088) [reply package.json:2]: rosie-prototype this name works right?

[the remark this answers is not in the document]
Comment (mattbeattie) [reply package.json:2]: Yep! `rosie-prototype` is fine.

**Pull request closed without merging**